specification/columns/tags.md

```diff
@@ -20,12 +22,13 @@ Provider-defined Tags additionally adhere to the following requirements:
 
 ## Provider-Defined vs. User-Defined Tags
 
-The following is an example of one user-defined tag and one provider-defined tag, respectively, with tag key, `foo`.  The first tag is user-defined and not prefixed. The second tag is provider-defined and prefixed with `acme/`, which the provider has specified as a reserved tag key prefix.
+The following is an example of one user-defined tag, one provider-defined tag, respectively, with tag key, `foo`, and one tag key without a supported value.  The first tag is user-defined and not prefixed. The second tag is provider-defined and prefixed with `acme/`, which the provider has specified as a reserved tag key prefix. The third tag with tag key, `baz`, does not support a value, so `null` is assigned as its value.
```

Comment:
> The third tag with tag key, `baz`, does not support a value, so `null` is assigned as its value.

If it does not *support* a value, does that mean it is label and should be "true"?

My semantics may be wrong, but to me, not *supporting* a value means "label", whereas not *having* a value means "null".

Reply:
I'm happy with everything else, but interested in your thinking here

Reply:
Sorry, just a mix up from the previous definition.  Fixed now.